docs/concepts/request-handling.md

```diff
@@ -0,0 +1,60 @@
+---
+id: request-handling
+title: Request Handling
+---
+
+In ZIO HTTP, request handling is the process of receiving an HTTP request, processing it, and generating an appropriate HTTP response. ZIO HTTP provides a functional and composable approach to request handling using the HttpApp type.
+
+The `HttpApp` type represents an HTTP application, which is essentially a function that takes an incoming Request and returns a Response. It is defined as type `HttpApp[R, E] = Request => ZIO[R, E, Response[E]]`, where `R` represents the required environment, `E` represents the potential error type, and `Response[E]` represents the response type.
+
+Here's an overview of the request handling process in ZIO HTTP:
+
+- **Defining an `HttpApp`**: To handle requests, you define one or more `HttpApp` values. Each `HttpApp` is responsible for processing a specific type of request or a specific path pattern.
+
+- **Request Matching**: The `Http.collect` or `Http.collectM` combinator is used to pattern match on the incoming `Request`. It allows you to define matching criteria based on HTTP method, path, headers, etc.
```

Comment:
Please note that the functions 'collect' and 'collectM' are no longer available and have been deprecated.